pkgs/os-specific/darwin/ios-cross/default.nix

```diff
@@ -1,61 +0,0 @@
-{ runCommand
-, lib
-, llvm
-, clang
-, binutils
-, stdenv
-, coreutils
-, gnugrep
-, buildPackages
-, hostPlatform
-, targetPlatform
-}:
-
-/* As of this writing, known-good prefix/arch/simulator triples:
```

Comment:
@elvishjerricco can you update `lib/systems/examples.nix` based on this? I tried to do it on my phone but GitHub makes it too hard.

Reply:
All 4 should have `useiOSPrebuilt = false;`

Reply:
I'm not actually sure what `crossSystem` is supposed to look like for this.